src/room.ts

```diff
@@ -204,19 +210,27 @@ export class Room extends EventEmitter implements Sayable {
       return null
     }
 
-    const memberList = this.parseMemberList(rawObj.MemberList)
-    const nameMap    = this.parseMap(rawObj.MemberList, 'name')
-    const aliasMap   = this.parseMap(rawObj.MemberList, 'alias')
+    if (rawObj.MemberList) {
+      const memberList = this.parseMemberList(rawObj.MemberList)
+      const nameMap    = this.parseMap(rawObj.MemberList, 'name')
+      const aliasMap   = this.parseMap(rawObj.MemberList, 'alias')
+      return {
+        id:         rawObj.UserName,
+        encryId:    rawObj.EncryChatRoomId, // ???
+        topic:      rawObj.NickName,
+        ownerUin:   rawObj.OwnerUin,
+
+        memberList,
+        nameMap,
+        aliasMap,
+      }
+    }
 
     return {
       id:         rawObj.UserName,
       encryId:    rawObj.EncryChatRoomId, // ???
       topic:      rawObj.NickName,
       ownerUin:   rawObj.OwnerUin,
-
-      memberList,
```

Comment:
If this.rawObj.MemberList is `undefined`, we should init **memberList** with `[]`.